scalastyle-config.xml

```diff
@@ -264,6 +264,19 @@ This file is divided into 3 sections:
     of Commons Lang 2 (package org.apache.commons.lang.*)</customMessage>
   </check>
 
+  <check customId="FileSystemGet" level="error" class="org.scalastyle.file.RegexChecker" enabled="true">
+    <parameters><parameter name="regex">FileSystem.get\([a-zA-Z_$][a-zA-Z_$0-9]*\)</parameter></parameters>
+    <customMessage><![CDATA[
+      Are you sure that you want to use "FileSystem.get(Configuration conf)"? If the input
+      configuration is not set properly, a default FileSystem instance will be returned. Please use
+      "FileSystem.get(URI uri, Configuration conf)" or "Path.getFileSystem()" instead.
```

Comment:
`Path.getFileSystem()` also takes a conf parameter?

Reply:
Right, thank you.